lib/settings-stuff.php

```diff
@@ -20,6 +20,8 @@
  */
 function available_php_versions() {
 	return array(
+		'8.1' => __( '8.1', 'jurassic-ninja' ),
```

Comment:
It'd be great to add 8.2 as well 👍

Reply:
Addressed in 6671beace23d5fb49e3004e6fedb6881156e58ab

Reply:
Thanks @oskosk 🙌